gammapy/scripts/image_pipe.py

```diff
@@ -2,56 +2,180 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 import logging
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.units import Quantity
+from astropy.io import fits
 from ..utils.scripts import get_parser
+from ..background import fill_acceptance_image
+from ..image import SkyMap, SkyMapCollection, disk_correlate
+from ..stats import significance
 
 __all__ = ['ImageAnalysis']
 
 log = logging.getLogger(__name__)
 
 
-def image_pipe_main(args=None):
-    parser = get_parser(ImageAnalysis)
-    parser.add_argument('config_file', type=str,
-                        help='Config file in YAML format')
-    # TODO: add option to dump the default config file
-    args = parser.parse_args(args)
-    analysis = ImageAnalysis.from_yaml(args.config_file)
-    analysis.run()
-
-
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
+
+    Parameters
+    ----------
+    empty_image : `~gammapy.image.SkyMap`
```

Comment:
Sorry @cdeil, I actually haven't read comment, because I was writing mine. But in the end we were saying the same thing: :+1:  to only pass the reference image and right now not allow to pass counts and bkg images.